codeclimate/client.go

```diff
@@ -0,0 +1,46 @@
+package codeclimate
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+const codeClimateApiHost string = "https://api.codeclimate.com/v1"
+
+type Client struct {
+	apiKey string
+}
+
+// TODO: Extend in the future to accept POST requests
+func (c *Client) makeRequest(path string) ([]byte, error) {
+	client := &http.Client{}
+
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s/%s", codeClimateApiHost, path), nil)
+
+	if err != nil {
+		log.Println(err)
+		return nil, err
+	}
+
+	req.Header.Add("Accept", `W/"application/vnd.api+json"`)
```

Comment:
I tried to ask it from people from Go world and no one answered me, but this was in the example, so TBH, it's copypaste, which I don't understand, but it works. I can try without it if it will work and then remove.